knative-operator/pkg/controller/knativekafka/replicastransform.go

```diff
@@ -9,10 +9,43 @@ import (
 	"k8s.io/client-go/kubernetes/scheme"
 )
 
+var (
+	KafkaHAComponents = []string{"kafka-ch-controller", "kafka-webhook", "kafka-controller-manager"}
+)
+
+func checkHAComponent(name string) bool {
+	for _, component := range KafkaHAComponents {
+		if name == component {
+			return true
+		}
+	}
+	return false
+}
+
 // replicasTransform makes kafka-ch-dispatcher keep its current replica count set by controller
 // based on vendor/knative.dev/operator/pkg/reconciler/knativeeventing/common/replicasenvvarstransform.go
```

Comment:
wasn't this added by @maschmid to allow the channel dispatcher to have its "own" replica, and not being reverted to what the YAMLs have, that are owned by the operator

Reply:
moved to its own function